pkgs/applications/networking/cluster/tgswitch/default.nix

```diff
@@ -10,7 +10,11 @@ buildGoPackage rec {
     sha256 = "sha256-6hErfI7LEJFgOoJR8IF9jTSBwqbQYeGiwdeJShqxVQ0=";
   };
 
-  goPackagePath = "github.com/warrensbox/tgswitch";
+  vendorSha256 = null;
+
+  ldflags = [ "-s" "-w" ];
+
+  doCheck = false;
```

Comment:
Is there any specific on why the tests are disable ? Can you add a comment for the same ?

Reply:
`buildGoPackage` disable checks by default.
```
doCheck = args.doCheck or false;
```

with `doCheck` enabled, there are some failed tests. So I disable checks for now.

Reply:
Maybe I can try to fix it.

Reply:
If you are not able to fix it, I guess you can just a comment explaining why the tests are failing... LGTM apart from this single comment. Thanks!